README.md

```diff
@@ -1,2 +1,48 @@
+
+:two_men_holding_hands: User-Friendly Interface :two_men_holding_hands:
+======================================================================
+<img align="right" height=160 width=410 style="margin-left: 20px" src="https://user-images.githubusercontent.com/52821952/64379078-d01ec680-d036-11e9-8f44-14010ae4b005.png">
+Interface of Pair Storm is pretty simple. If you were using integrated development environment like Visual Studio, Qt Creator, CLion and etc before, work with Pair Storm will not be a problem for you. The most used tools is always in sight. If you want access to advanced settings use the relevant menu.
+ 
+:speech_balloon: Embedded Chat :speech_balloon:
+===============================================
+<img align="left" width=207 style="margin-right: 20px" src="https://user-images.githubusercontent.com/52821952/64411882-dcd30700-d096-11e9-9460-9deff5e4c14c.png">
+Pair Storm IDE has embedded chat. It helps to communicate with another programmers during development process. In chat can hang out many users at the same time. All messages are saving in local database, which is unique for each project. Embedded chat is secure. Get access to the message history can only authorized users, which work have appropriate premissions.
+
+:hammer: Refactoring Tools :hammer:
+===================================
+<img align="right" width=220 style="margin-left: 20px" src="https://user-images.githubusercontent.com/52821952/64385117-dc0e8680-d03e-11e9-993a-bc40d504a335.png">
+IDE gives users many useful tools, which make development process easier and faster. For example, Pair Storm has embedded class generator. Using it programmer can create .h and .cpp files, define new methods in both file and etc. 
+
+Also Pair Storm have many useful features, which help programmer to write good code: auto tabulation, auto completion and etc.
+
+:high_brightness: Syntax Highlighting :high_brightness:
+=======================================================
+<img align="left" width=500 style="margin-right: 20px" src="https://user-images.githubusercontent.com/52821952/64383380-70c4b480-d03e-11e9-8eed-fe0789acc060.png">
+Embedded code editor displays text in different colors. This helps to make source code more readable, simple for comprehension. Also highlighter recognize invalid construction. This lets find mistakes errors in source code and fix them before compilation.
+
+
+:heavy_plus_sign: Comments :heavy_plus_sign:
+============================================
+<img align="right" width=480 style="margin-left: 20px" src="https://user-images.githubusercontent.com/52821952/64407659-8e6d3a80-d08d-11e9-94df-462c6e6eda4c.png">
+Pair Storm supports comments in Github style. This helps to realize conception of pair programing, when one person write source code and another - review it right away. All comments are saving in local database, which is unique for each project. User can write comments to either line of code.
+
```

Comment:
Pair Storm allows users to leave Github styled comments in the code. Commenting is significant part of the pair programming conception. Thus, in context of working with code, there are now two categories of users: developers and commenters. All comments also get's saved in the project' database. Comments are bounded to lines of code, which means that registered user can comment out certain expressions separately.